Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -60,6 +62,9 @@ Newly fetched incidents or detections will be mirrored in the chosen direction.
  - When *mirroring in* incidents from CrowdStrike Falcon to XSOAR:
    - For the `tags` field, tags can only be added from the remote system.
    - When enabling the *Close Mirrored XSOAR Incident* integration parameter, the field in CrowdStrike Falcon that determines whether the incident was closed is the `status` field.
+   - In case the **look-back** parameter is initialized with a certain value and during a time that incidents were fetched, if changing 
```

Comment:
```suggestion
   - In case the *look-back* parameter is initialized with a certain value and during a time that incidents were fetched, if changing 
```